src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampRequestNetstandard21Wrapper.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+#if IS_SIGNING_SUPPORTED && IS_CORECLR
+using System.Net.Http;
+using System.Net.Http.Headers;
+#endif
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace NuGet.Packaging.Signing
+{
+#if IS_SIGNING_SUPPORTED && IS_CORECLR
+    internal class Rfc3161TimestampRequestNetstandard21Wrapper : IRfc3161TimestampRequest
+    {
+        private static readonly HttpClient HttpClient = new HttpClient();
+        private readonly System.Security.Cryptography.Pkcs.Rfc3161TimestampRequest _rfc3161TimestampRequest;
+
+        public Rfc3161TimestampRequestNetstandard21Wrapper(
+                byte[] messageHash,
+                HashAlgorithmName hashAlgorithm,
+                Oid requestedPolicyId,
+                byte[] nonce,
+                bool requestSignerCertificates,
+                X509ExtensionCollection extensions)
```

Comment:
Indent 1 level (4 spaces not 8) from line 22.

Reply:
Thanks! Fixed.